src/Bridge/Twig/Extension/EnumExtension.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Greg0ire\Enum\Bridge\Twig\Extension;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class EnumExtension extends \Twig_Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return [
+            new \Twig_SimpleFunction('enum_label', [$this, 'label']),
+        ];
+    }
+
+    /**
```

Comment:
Oh right. Makes sense. What would be awesome would be not to have to specify the enum class name, like this `{{ enum_label(member, 'accessLevel') }}`. That would mean that the twig extension would be able to retrieve the correct enum class for a given field, which could be done by relying on an annotation. That would also mean not using a filter in this case. What do you think?